### ISSUE TITLE
Admin overview of all created rooms

### ISSUE THREAD

**Issue by schellmann** [sualko/cloud_bbb#108]
**Is your feature request related to a problem? Please describe.**
When a user has a problem with a room, it is sometimes difficult to see if there is really a technical problem or the user has set the wrong settings.

**Describe the solution you'd like**
A separate overview for admins in which all created rooms and settings are listed.
Furthermore, it would be great if the admin could change the settings of individual rooms.

**Comment by Smith-fms** [comment]
Even with this, an overview of how many users in all rooms.

**Comment by b3** [comment]
That would be really awesome and will simplify the work of admin **a lot** in numerous different situations. That could also simplify planning.

**Comment by sualko** [comment]
Pull requests welcome :wink:

**Comment by skynick11** [comment]
Yes, Greenlight has the room list (and with management features), but the read-only list of the rooms+status (started/stopped)+the number of users in each started room is much better than nothing.

But Greenlight also has the overview feature for recordings (as a separate list), and the possibility to manage recordings can even more important than the possibility to edit user's room settings by admin "from the one point", because they can use too much disk space at the BBB server.

I don't know what is better - to have a separate overview of server recordings or to have a combined overview of rooms and recordings?
